tests/testthat/test-ggcoef_model.R

```diff
@@ -124,3 +128,17 @@ test_that("example of ggcoef_model", {
 
 
 })
+
+test_that("ggcoef_model works with tieders not returning p-values", {
+  mod <- lm(Sepal.Width ~ Species, iris)
```

Comment:
Double check if tests can be run

```suggestion
  skip_if_not_installed("broom.helpers")
  skip_if_not_installed("scagnostics")

  mod <- lm(Sepal.Width ~ Species, iris)
```